lib/modules/manager/bundler/artifacts.ts

```diff
@@ -93,7 +93,10 @@ export async function updateArtifacts(
 
     const commands: string[] = [];
 
-    if (config.isLockFileMaintenance) {
+    if (
+      updatedDeps.some((dep) => dep.updateType === 'lockFileMaintenance') ||
+      config.isLockFileMaintenance
+    ) {
```

Comment:
@rarkins i think i'll fix it by properly fill `isLockFileMaintenance` on grouping, which is the only case where this can be false because of the first update in group isn't a lockfile update

Reply:
- #33971